cpp/ql/test/library-tests/dataflow/taint-tests/taint.cpp

```diff
@@ -192,7 +192,7 @@ void *memcpy(void *dest, void *src, int len);
 void test_memcpy(int *source) {
 	int x;
 	memcpy(&x, source, sizeof(int));
-	sink(x); // $ ast=192:23 MISSING: ir SPURIOUS: ast=193:6
+	sink(x); // $ ast=192:23 ir SPURIOUS: ast=193:6
```

Comment:
There's now an `ir` result, but is it from the correct source (line 192) or the wrong one (line 193)? Same in `test_mempcpy` and `test_memccpy` further down.

Reply:
Good point. Currently, we only allow a `line:column` annotation on a sink when there's more than one source for it, but for cases such as this, we actually want to highlight the exact source even though there should only be one.

Reply:
You're gonna have to trust me when I say that it's the correct one for now 😄 I've made a [PR](https://github.com/github/codeql/pull/5417) with the changes to the inline expectation library so that we can provide "optional" values. Once that PR gets merged (assuming the other languages are okay with it 🤞) we can revisit this annotation.